### ISSUE TITLE
Component is not accessible to keyboard users

### ISSUE THREAD

**Issue by aarongustafson** [samhirtarif/react-audio-recorder#121]
**Describe the bug**
As it is currently built, this component is not accessible to keyboard users. Controls cannot take focus, accessible names & roles are not managed, etc.

**To Reproduce**
1. Try to bring focus to the component using your keyboard
2. Try to interact with the component with a keyboard after opening it

**Expected behavior**
All buttons should be focusable and be interactive using both keyboard and mouse.

**Package info (please complete the following information):**
 - 2.2.0
